flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendRunTest.java

```diff
@@ -285,7 +345,7 @@ public static void verifyCliFrontend(
         RunTestingCliFrontend testFrontend =
                 new RunTestingCliFrontend(
                         configuration,
-                        clusterClientServiceLoader,
+                        new DefaultClusterClientServiceLoader(),
```

Comment:
Not sure why this change is needed? I see `new DefaultClusterClientServiceLoader` other places too, is it possible to do it once per CLI?

Reply:
This was like this before, I just removed one unused `verifyCliFrontend` method. Anyways, this can be moved to a static constant of the `RunTestingCliFrontend` nested class.